### PR TITLE
Add categories to database

### PR DESCRIPTION
**Pull request by michaelsilver**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,156 @@
-# Python
-.eggs
-__pycache__
-.pytest_cache
-Dormspam.egg-info
+### Project ###
```
Comment (jackcook) [inline .gitignore]: Wait why did you add all of this lol

Comment (michaelsilver) [reply .gitignore]: Because I think some `.pyc` files or something were showing up tracked by git so I wanted to make sure _nothing_ unwanted from python or osx would be tracked in the future. It's autogenerated from https://www.gitignore.io/
